app/models/db_client.py

```diff
@@ -27,7 +27,7 @@ def create_entity(data):
     session.close()
 
     # return the id to the caller
-    return data.id
+    return data
```

Comment:
I'd prefer to return ID here. What's the use case for returning the whole payload?

Reply:
Couple of reasons for that:
1. The frontend will need the entire payload to render the page; so if the frontend creates a scenario and just gets the ID, the frontend will need to make 2 requests to render the page, first for the POST, and then a GET for the scenario
2. I don't see a reason to NOT return the newly created object. There are certain things that will only be populated once the object is created (`id` is just one example, but other things like `scene_ids`, `is_published` etc. are not part of the initial payload, but are part of the newly created object).

Is there an explicit reason for only wanting the id to be returned?

Reply:
I don't think #1 is always true. If I am creating a new asset, I upload an image and give a name/description and then ship it off - I don't want the whole payload back. I will, at a different time, request the asset.

In fact, isn't the flow I described above the way it's going to work when creating almost everything? You won't immediately use it, we will have a different flow for creating stuff.

The main reason to not give back the whole payload is that it can increase the payload size and slow down the endpoints. In fact, we only want to return a status code on a POST request imo.

Reply:
You pointed out the only instance when 1 isn't true. In other cases, it will be true. Create a new scenario? -> GET scenario to render it. Create a new scene? -> GET scene to render it. Create a new object? -> GET object to render it on the scene. Add text to a scene? GET text to render it and so on...

It will be significantly slower to do 2 requests than returning the entire payload. The entire payload is just a JSON obj with less than 20 keys.

Reply:
I still don't think that's true. It's hard to communicate this when we haven't finalized admin design but my idea of the "scenario/scene" editors is that you just have a dropdown of objects, text, and assets to use. Even with making scenes and scenarios, you should be able to create one independent of immediately loading it into the scenario.

But fair point, the payload isn't massive so perhaps it's not a big deal.